File: node/mongo.js
const MongoClient = require('mongodb').MongoClient;
const assert = require('assert');
const config = require('../config.json');

class MongoHelper {
    async insertDocuments(col, docs, callback){
            // Connection URL
        const url = config.mongo.connection_string;
        return new Promise(async (resolve, reject) => {
            const client = await MongoClient.connect(url, {useUnifiedTopology: true});
            const db = await client.db(config.mongo.db_name);
            const collection = await db.collection(col);
            await collection.insertMany(docs, function(err, result) {
                if(err) reject(err);
                console.log("document inserted into the collection");
                client.close();
                resolve(result)
            });
                
        });
    }  
}


module.exports = MongoHelper;

File: src/mongo.py
from pymongo import MongoClient
import os



class MongoHelper(object):

    def __init__(self, db_name="datapare"):
        self.db_name = db_name

    def create_client(self):
        client = MongoClient('mongodb://{0}:{1}@{2}'.format(os.getenv("MONGO_USER"), os.getenv("MONGO_PASSWORD"), os.getenv("MONGO_HOST")))
        return client[self.db_name]

    def insert(self, collection: str, data: dict) -> None:
        db = self.create_client()
        _collection = db[collection]
        _collection.insert(data, check_keys=False)

    def query(self,  collection: str, data: dict) -> None:
        db = self.create_client()
        _collection = db[collection]
        return _collection.find(data)